jobs.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: simple 
+---
+
+# Desenvolupador backend
+
+## Sobre Coopdevs
+
+Coodevs és una entitat que pretén aportar software lliure a l'economia social i solidària. Sorgida des de professionals del sector tecnològic de Barcelona amb inquietuds socials, proporciona amplis coneixements tècnics i metodologies de treball en aquest àmbit econòmic. Entre molts altres projectes, impulsem Katuma, formem part de l'equip de desenvolupament d'Open Food Network, desenvolupem TimeOverflow i tenim a projectes cooperatius
```

Comment:
`s/Coodevs/Coopdevs/`

Link to Katuma, OFN and TO?

`i tenim a projectes cooperatius`, I don't understand this

Reply:
tenim altres projectes cooperatius?

Reply:
Em referia a altres clients però ho he tret i menys lio